### PR TITLE
Feature/ibc UI

### PR DESCRIPTION
**Pull request by Goldselleruk**
#👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-com/chain-desktip-wallet/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`yarn build`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`yarn test`)
- [ ] Have you checked your code formatting is correct? (`yarn lint:js`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`yarn audit`)
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [x] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-com/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-com/chain-desktop-wallet/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-com/chain-desktop-wallet/blob/master/CONTRIBUTING.md#developer-certificate-of-origin).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: src/service/rpc/NodeRpcService.ts [hunk 1]
```diff
@@ -288,6 +297,31 @@ export class NodeRpcService implements INodeRpcService {
       response.data.pagination.next_key,
```
Comment (Goldselleruk) [inline src/service/rpc/NodeRpcService.ts:297]: Gdzie gold

**Review by Goldselleruk — COMMENTED** [review]
Gdzie gold

File: src/service/WalletService.ts [hunk 2]
```diff
@@ -70,7 +70,7 @@ import {
   WithdrawStakingRewardRequest,
 } from './TransactionRequestModels';
 import { FinalTallyResult } from './rpc/NodeRpcModels';
-import { sleep } from '../utils/utils';
+import { capitalizeFirstLetter, sleep } from '../utils/utils';
```
Comment (Goldselleruk) [inline src/service/WalletService.ts:73]: Gdzie gold

File: src/service/WalletService.ts [hunk 3]
```diff
@@ -70,7 +70,7 @@ import {
...
 
 class WalletService {
```
Comment (Goldselleruk) [inline src/service/WalletService.ts:75]: Gdzie gold

File: src/service/WalletService.ts [hunk 4]
```diff
@@ -673,8 +673,14 @@ class WalletService {
...
     await Promise.all(
       assets.map(async asset => {
-        const baseDenomination = currentSession.wallet.config.network.coin.baseDenom;
+        const baseDenomination =
+          asset.assetType !== UserAssetType.IBC
+            ? currentSession.wallet.config.network.coin.baseDenom
+            : `ibc/${asset.ibcDenomHash}`;
+
         try {
+          // eslint-disable-next-line no-console
```
Comment (Goldselleruk) [inline src/service/WalletService.ts:682]: Gdzie gold

File: src/service/WalletService.ts [hunk 5]
```diff
@@ -673,8 +673,14 @@ class WalletService {
 
     await Promise.all(
       assets.map(async asset => {
-        const baseDenomination = currentSession.wallet.config.network.coin.baseDenom;
+        const baseDenomination =
```
Comment (Goldselleruk) [inline src/service/WalletService.ts:676]: Gdzie gold

File: src/service/WalletService.ts [hunk 6]
```diff
@@ -673,8 +673,14 @@ class WalletService {
 
     await Promise.all(
       assets.map(async asset => {
-        const baseDenomination = currentSession.wallet.config.network.coin.baseDenom;
```
Comment (Goldselleruk) [inline src/service/WalletService.ts:676]: Gdzie gold

**Review by Goldselleruk — COMMENTED** [review]
Gdzie gold

**Comment by crypto-3dd** [comment]
Is there a reason you are pushing an unfinished PR branch that you didn't start. Please read the contributions guidelines

**Pull request closed without merging**